lib/index.js

```diff
@@ -58,6 +59,11 @@ function createAssets(){
   assets.push(path.join(__dirname, '../plug/requirements/vcs.py'));
   assets.push(path.join(__dirname, '../plug/requirements/__init__.py'));
 
+  assets.push(path.join(__dirname, '../plug/pytoml/__init__.py'));
+  assets.push(path.join(__dirname, '../plug/pytoml/core.py'));
+  assets.push(path.join(__dirname, '../plug/pytoml/parser.py'));
+  assets.push(path.join(__dirname, '../plug/pytoml/writer.py'));
+
   return assets;
 }
 
```

Comment:
please update the `catch` in `getDependencies` to not always print `pip install ...`